misc/db_schema.yaml

```diff
@@ -1599,7 +1599,7 @@ vminfo:
     - { Field: id, Type: int(11), 'Null': false, Extra: auto_increment }
     - { Field: device_id, Type: int(11), 'Null': false, Extra: '' }
     - { Field: vm_type, Type: varchar(16), 'Null': false, Extra: '', Default: vmware }
-    - { Field: vmwVmVMID, Type: int(11), 'Null': false, Extra: '' }
+    - { Field: vmwVmVMID, Type: char(36), 'Null': false, Extra: '' }
```

Comment:
Why is this changing? Again vmware seems to expect a numeric ID which was the same for libvirt before.

Reply:
_only_ running VMs even have an integer ID with libvirt. Even worse, the ID is ephemeral. Here is what happens when I start and stop a VM on a real machine:
```
$ virsh -c 'qemu:///system' list --all
 Id    Name                           State
----------------------------------------------------
 --snip--
 16    test                           running
 -     alarm-aarch64                  shut off

$ virsh -c 'qemu:///system' shutdown test
Domain test is being shutdown

$ virsh -c 'qemu:///system' list --all 
 Id    Name                           State
----------------------------------------------------
--snip--
 -     alarm-aarch64                  shut off
 -     test                           shut off

$ virsh -c 'qemu:///system' start test
Domain test started

$ virsh -c 'qemu:///system' list --all
 Id    Name                           State
----------------------------------------------------
--snip--
 17    test                           running
 -     alarm-aarch64                  shut off
```
As you can see the ID changed, which would cause a lot of garbage events in librenms currently. You are correct that this will affect VMWare, my mistake. I wonder if I need to make a new column for the UUID?